src/v/kafka/group_probe.h

```diff
@@ -12,18 +12,26 @@
 #pragma once
 
 #include "config/configuration.h"
+#include "kafka/server/member.h"
+#include "kafka/types.h"
+#include "model/fundamental.h"
+#include "model/namespace.h"
 #include "prometheus/prometheus_sanitize.h"
+#include "ssx/metrics.h"
 
+#include <absl/container/node_hash_map.h>
+#include <absl/container/node_hash_set.h>
 #include <seastar/core/metrics.hh>
 
 namespace kafka {
 class group_offset_probe {
 public:
     explicit group_offset_probe(model::offset& offset) noexcept
-      : _offset(offset) {}
+      : _offset(offset)
+      , _public_metrics(ssx::public_metrics_handle) {}
 
     void setup_metrics(
-      const kafka::group_id& group_id, const model::topic_partition& tp) {
+      const kafka::group_id& group_id, const model::topic_partition& tp, model::topic_namespace& tp_ns) {
```

Comment:
> I think I followed it through correctly, but isn't this the `tp_ns` of the `model::kafka_group_nt`?

I'll check, but I also think we can remove namespace from this metric since using the consumer group API guarantees the namespace will always be kafka. I'll make a comment in the RFC document to collect opinions.

Reply:
Yeah, let's just remove namespace.

Reply:
Done. Changes were pushed to a separate PR at https://github.com/redpanda-data/redpanda/pull/5349